gt-scheduler/website
# Feedback Close Button Fix

### Summary

Resolves #180 


Fixes the feedback window close button resetting the submission page

### Checklist

- [x]  Make sure the x button on the top right closes the session window
- [x] Make sure that the close button on the bottom center ends the session and closes the session window
- [x] Upon reopening the feedback window after the close button, make sure a new session starts


### How to Test

Open the Feedback window, enter a rating and text, then hit "submit" and then "close" and reopen the window. The submission form should be reset.
